src/test/util/net.h

```diff
@@ -30,4 +30,27 @@ struct ConnmanTestMsg : public CConnman {
     bool ReceiveMsgFrom(CNode& node, CSerializedNetMsg& ser_msg) const;
 };
 
+constexpr NetPermissionFlags ALL_NET_PERMISSION_FLAGS[]{
+    NetPermissionFlags::PF_NONE,
+    NetPermissionFlags::PF_BLOOMFILTER,
+    NetPermissionFlags::PF_RELAY,
+    NetPermissionFlags::PF_FORCERELAY,
+    NetPermissionFlags::PF_NOBAN,
+    NetPermissionFlags::PF_MEMPOOL,
+    NetPermissionFlags::PF_ADDR,
+    NetPermissionFlags::PF_DOWNLOAD,
+    NetPermissionFlags::PF_ISIMPLICIT,
+    NetPermissionFlags::PF_ALL,
+};
+
+constexpr ServiceFlags ALL_SERVICE_FLAGS[]{
```

Comment:
nit: wrong order - `ALL_SERICE_FLAGS` should go before `ALL_NET_PERMISSION_FLAGS` see original PR: https://github.com/bitcoin/bitcoin/pull/20789/files#diff-465d5e9d944d416f0175356866bb5d04c79ab11f46510558df59496964f79c71R42


note: it's partial due to missing 26304b4100201754fb32440bec3e3b78cd3f0e6d - #19316

Reply:
But bitcoin#21843's already labelled as partial.... see 021ada38a736f390e0e167c1a6cee81bddf34b2b

If you meant bitcoin#20789, I'll mark that as partial as well since that's what the review seems to be about and fix the ordering as well.